### PR TITLE
Demo: Colored depth map values correction by using minimum range

### PR DESCRIPTION
**Pull request by grigorovici-andreea**
Added minimum extremity range in camera details.
Used the minimum extremity range to correct the values in colored depth map.

Solves #185 

Signed-off-by: Andreea Grigorovici <andreea.grigorovici@analog.com>

### CODE REVIEW

File: sdk/include/aditof/camera_definitions.h [hunk 1]
```diff
@@ -56,9 +56,16 @@ struct CameraDetails {
     ConnectionType connection;
 
     /**
-     * @brief The range of the camera in mm for the operating mode
+     * @brief The maximum extremity of the range of the camera in mm for the
+     * operating mode
      */
-    int range;
+    int rangeMax;
```
Comment (dNechita) [inline sdk/include/aditof/camera_definitions.h]: how about maxDepth and minDepth? It's more descriptive, makes it clear that this is for depth and not IR.

File: sdk/include/aditof/camera_definitions.h [hunk 2]
```diff
@@ -56,9 +56,16 @@ struct CameraDetails {
     ConnectionType connection;
 
     /**
-     * @brief The range of the camera in mm for the operating mode
+     * @brief The maximum extremity of the range of the camera in mm for the
```
Comment (dNechita) [inline sdk/include/aditof/camera_definitions.h]: The maximum depth the camera can measure in the current operating mode. Distance is in millimeters.

File: sdk/src/camera_96tof1.cpp [hunk 3]
```diff
@@ -61,9 +61,12 @@ aditof::Status Camera96Tof1::setMode(const std::string &mode,
 
     LOG(INFO) << "Chosen mode: " << mode.c_str();
 
-    std::vector<std::pair<std::string, int>> modeRanges = {
+    std::vector<std::pair<std::string, int>> modeRangesMax = {
```
Comment (dNechita) [inline sdk/src/camera_96tof1.cpp]: You could create a struct which can describe a range. Make it internal to this class. And have just one array. You won't need to use the second find_if either.

File: sdk/include/aditof/camera_definitions.h (hunk 2 above)
Comment (grigorovici-andreea) [reply sdk/include/aditof/camera_definitions.h]: done, thanks

File: sdk/include/aditof/camera_definitions.h (hunk 1 above)
Comment (grigorovici-andreea) [reply sdk/include/aditof/camera_definitions.h]: done, thanks

File: sdk/src/camera_96tof1.cpp (hunk 3 above)
Comment (grigorovici-andreea) [reply sdk/src/camera_96tof1.cpp]: done, thanks

File: bindings/python/aditofpython.cpp [hunk 4]
```diff
@@ -53,7 +53,7 @@ PYBIND11_MODULE(aditofpython, m) {
         .def_readwrite("mode", &aditof::CameraDetails::mode)
         .def_readwrite("frameType", &aditof::CameraDetails::frameType)
         .def_readwrite("connection", &aditof::CameraDetails::connection)
-        .def_readwrite("range", &aditof::CameraDetails::range)
+        .def_readwrite("range", &aditof::CameraDetails::rangeMax)
```
Comment (dNechita) [inline bindings/python/aditofpython.cpp]: This line should be: .def_readwrite("maxDepth", &aditof::CameraDetails::maxDepth). There's no rangeMax anymore. The python bindings shouldn't even compile anymore.
Also, can you create another binding for minDepth as well?

Comment (grigorovici-andreea) [reply bindings/python/aditofpython.cpp]: totally forgot about bindings. working on it now.

Comment (grigorovici-andreea) [reply bindings/python/aditofpython.cpp]: changed. thank you.

**Pull request merged**